docs/.gitignore

```diff
@@ -1,3 +1,4 @@
 yarn-error.log
 node_modules
 .DS_Store
+docs/.vuepress/dist
```

Comment:
@dylburger funny, I added this to one of my more recent branches too. Got tired of `git clean`'ing every time.